### ISSUE TITLE
unhashable type: 'LineString'

### ISSUE THREAD

**Issue by denadai2** [gboeing/osmnx#10]
While I was trying to repeat this: https://github.com/gboeing/osmnx/blob/master/examples/04-example-simplify-network.ipynb, I have this issue:

    import osmnx as ox
    G = ox.graph_from_place('caneva, Italy')
    ox.plot_graph(G, node_color='b', node_zorder=3)
    # simplify the network
    G2 = G.copy()
    G2 = ox.simplify_graph(G2)

result:

    TypeError                                 Traceback (most recent call last)
    <ipython-input-61-b9e77563d9b6> in <module>()
          1 # simplify the network
          2 G2 = G.copy()
    ----> 3 G2 = ox.simplify_graph(G2)

    /Library/Frameworks/Python.framework/Versions/3.4/lib/python3.4/site-packages/osmnx/osmnx.py in simplify_graph(G_, strict)
       2373         for key in edge_attributes:
       2374             # don't touch the length attribute, we'll sum it at the end
    -> 2375             if len(set(edge_attributes[key])) == 1 and not key == 'length':
       2376                 # if there's only 1 unique value in this attribute list, consolidate it to the single value (the zero-th)
       2377                 edge_attributes[key] = edge_attributes[key][0]

    TypeError: unhashable type: 'LineString'

Python3, shapely 1.6.

**Comment by gboeing** [comment]
Yes, this is because you are trying to simplify a graph that has already been simplified. By default, graph_from_place performs simplification before returning the graph to you (it has a `simplify` argument with default value = True). If you want to simplify manually after creating the graph, turn simplify off when you call graph_from_place, like this:

```
import osmnx as ox
G = ox.graph_from_place('Caneva, Italy', simplify=False)
ox.plot_graph(G)
G2 = ox.simplify_graph(G.copy())
ox.plot_graph(G2)
```

I will create a clearer exception message when trying to simplify an already simplified graph so that this is unambiguous.

**Comment by gboeing** [comment]
Ok, new error message has been pushed and is available in the latest beta version, 0.1b2. Available via `pip install osmnx --upgrade`.

**Comment by denadai2** [comment]
awesome! Thanks for the velocity :))
